src/backend/distributed/commands/function.c

```diff
@@ -43,9 +62,20 @@ Datum
 create_distributed_function(PG_FUNCTION_ARGS)
 {
 	RegProcedure funcOid = PG_GETARG_OID(0);
+
+	text *distributionArgumentNameText = PG_GETARG_TEXT_P(1);
```

Comment:
Maybe this should also be optional, such that you can use create_distributed_function to just use a function in queries without other magic?

Reply:
Yeah, that also sounds good. Let me think a bit more

Reply:
I've done that, looks good to me as well.